.github/workflows/startManualRelease.yml

```diff
@@ -27,7 +27,6 @@ jobs:
           skip-on-empty: false
           output-file: false
           pre-commit: ./scripts/bump-lerna-version.js
-          tag-prefix: ''
```

Comment:
This is being ignored in favor of the default value ("v"), so I removed it as well.